mscore/timeline.h

```diff
@@ -28,6 +28,8 @@
 
 namespace Ms {
 
+#include<QGraphicsItem>
```

Comment:
Sure this is needed and not already in all.h? No space between `#include` and `<`?